jobspec/parse.go

```diff
@@ -537,6 +538,22 @@ func parseTasks(jobName string, taskGroupName string, result *[]*structs.Task, l
 					return err
 				}
 			}
+
+			// Instantiate a driver to validate the configuration
+			d, err := driver.NewDriver(
```

Comment:
You can create a new package so that structs can depend on that which in turn depends on driver but driver wont depend on it